tests/integration/components/callout-banner-test.js

```diff
@@ -0,0 +1,32 @@
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+module('Integration | Component | callout-banner', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`
+      <CalloutBanner>
+        I have an announcement to make
+      </CalloutBanner>
+    `);
+
+    assert
+      .dom('.callout-banner')
+      .exists()
+      .hasText('I have an announcement to make');
```

Comment:
`hasText()` is a stronger assertion than `.exists()` is. This means, we can remove the latter (a weaker assertion) and still have the same test coverage.

(Aside: Here, the terms "strong" and "weak" come from math when writing a proof.)

```suggestion
      .dom('[data-test-callout-banner]')
      .hasText('I have an announcement to make');
```

I'd also recommend using a `data-test-*` instead.

Related: https://github.com/ember-learn/ember-website/pull/925/files#r832893092